src/main/java/org/csanchez/jenkins/plugins/kubernetes/NonConfigurableKubernetesCloud.java

```diff
@@ -25,6 +26,14 @@ public void addTemplate(PodTemplate template) {}
     @Override
     public void removeTemplate(PodTemplate template) {}
 
+    @Override
+    public boolean hasManagePermission() {
+        return false;
+    }
+
+    @Override
+    public void checkManagePermission() throws AccessDeniedException {}
```

Comment:
Looks wrong, but all methods affecting templates are already no-op in this class I guess.